Algorithm.Framework/Execution/NullExecutionModel.cs

```diff
@@ -13,21 +13,16 @@
  * limitations under the License.
 */
 
-using QuantConnect.Algorithm.Framework.Portfolio;
 using QuantConnect.Data.UniverseSelection;
 
 namespace QuantConnect.Algorithm.Framework.Execution
 {
     /// <summary>
     /// Provides an implementation of <see cref="IExecutionModel"/> that does nothing
     /// </summary>
-    public class NullExecutionModel : IExecutionModel
+    public class NullExecutionModel : ExecutionModel
     {
-        public virtual void OnSecuritiesChanged(QCAlgorithmFramework algorithm, SecurityChanges changes)
-        {
-        }
-
-        public virtual void Execute(QCAlgorithmFramework algorithm, IPortfolioTarget[] targets)
+        public override void OnSecuritiesChanged(QCAlgorithmFramework algorithm, SecurityChanges changes)
```

Comment:
Need to add `Execute` method and remove `OnSecuritiesChanged` -- otherwise will throw `NotImplementedException` from base class

Reply:
fixed.